Packages/com.unity.inputsystem/Documentation~/Interactions.md

```diff
@@ -79,6 +79,8 @@ If multiple Interactions are present on a single Binding or Action, then the Inp
 
 At any one time, only one Interaction can be "driving" the action (that is, it gets to determine the action's current [`phase`](../api/UnityEngine.InputSystem.InputAction.html#UnityEngine_InputSystem_InputAction_phase)). If an Interaction higher up in the stack cancels, Interactions lower down in the stack can take over.
 
+Note that the order of interactions can effect which interaction is passed to your callback function. For example, an action with Tap, Multi Tap and Hold interactions will have different behaviour when the interactions are in a different order, such as Hold, Multi Tap and Tap. If you get unexpected behaviour, you may need to experiment with a different ordering.
```

Comment:
Thanks for adding this, I think its a valuable example given the associated bug report.

Minor: I would suggest hyperlinking the Tap, Multi Tap and Hold interactions to their corresponding script pages for reference.